### PR TITLE
Newly joined Seed don't have sharding structure until recv VCDSBlockMessage

### PR DESCRIPTION
**Pull request by sandipbhoir**
## Description
This PR fixes the issue : https://github.com/Zilliqa/Issues/issues/453

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] small-scale cloud test

### CODE REVIEW

File: src/libLookup/Lookup.cpp [hunk 1]
```diff
@@ -3143,6 +3150,21 @@ void Lookup::ComposeAndSendGetDirectoryBlocksFromSeed(const uint64_t& index_num,
...
 
+void Lookup::ComposeAndSendGetShardingStructureFromSeed() {
+  LOG_MARKER();
+  bytes message = {MessageType::LOOKUP,
+                   LookupInstructionType::GETSHARDSFROMSEED};
+
+  if (!Messenger::SetLookupGetShardsFromSeed(
+          message, MessageOffset::BODY,
+          m_mediator.m_selfPeer.m_listenPortHost)) {
+    LOG_GENERAL(WARNING, "Messenger::SetLookupGetDirectoryBlocksFromSeed");
```
Comment (ansnunez) [inline src/libLookup/Lookup.cpp]: Wrong message

File: src/libLookup/Lookup.cpp [hunk 2]
```diff
@@ -1203,82 +1214,78 @@ bool Lookup::ProcessGetStateDeltaFromSeed(const bytes& message,
...
+    return false;
+  }
 
-  // if (!VerifySenderNode(GetLookupNodes(), lookupPubKey)) {
-  //   LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
-  //             "The message sender pubkey: "
-  //                 << lookupPubKey << " is not in my lookup node list.");
-  //   return false;
-  // }
+  if (!VerifySenderNode(GetLookupNodes(), lookupPubKey)) {
```
Comment (ansnunez) [inline src/libLookup/Lookup.cpp]: Should this be `GetSeedNodes()`?

File: src/libLookup/Lookup.cpp [hunk 3]
```diff
@@ -1203,82 +1214,78 @@ bool Lookup::ProcessGetStateDeltaFromSeed(const bytes& message,
...
 
-  // if (shardingStructureVersion != SHARDINGSTRUCTURE_VERSION) {
-  //   LOG_GENERAL(WARNING, "Sharding structure version check failed. Expected:
-  //   "
-  //                            << SHARDINGSTRUCTURE_VERSION
-  //                            << " Actual: " << shardingStructureVersion);
-  //   return false;
-  // }
+  if (shardingStructureVersion != SHARDINGSTRUCTURE_VERSION) {
+    LOG_GENERAL(WARNING, "Sharding structure version check failed. Expected:"
```
Comment (ansnunez) [inline src/libLookup/Lookup.cpp]: Can use this instead:
`LOG_CHECK_FAIL("Sharding structure version", shardingStructureVersion, SHARDINGSTRUCTURE_VERSION);`

File: src/libLookup/Lookup.cpp (hunk 2 above)
Comment (sandipbhoir) [reply src/libLookup/Lookup.cpp]: Its fine. need to verify if sender is from lookup node list.

**Pull request merged**